wallet/how-to/send-transactions/send-batch-transactions.md

```diff
@@ -0,0 +1,204 @@
+---
+description: Send atomic batch transactions using `wallet_sendCalls`.
+---
+
+# Send batch transactions
+
+You can send and manage batch transactions in MetaMask, using the methods specified by
+[EIP-5792](https://eips.ethereum.org/EIPS/eip-5792):
+
+- `wallet_getCapabilities` - Query whether support for atomic batch transactions is available.
+- `wallet_sendCalls` - Submit multiple transactions to be processed atomically by MetaMask.
+- `wallet_getCallsStatus` - Track the status of your transaction batch.
+
+## About atomic batch transactions
+
+When a dapp requests to submit a batch of transactions atomically, MetaMask may prompt users to upgrade their externally owned account (EOA) to a [MetaMask delegator account](https://docs.gator.metamask.io/concepts/delegator-accounts).
+If the user accepts, MetaMask proceeds to upgrade the account and process the request as a single atomic transaction as specified by [EIP-7702](https://eips.ethereum.org/EIPS/eip-7702).
+
+:::note Delegator accounts
+MetaMask delegator accounts are [ERC-4337](https://eips.ethereum.org/EIPS/eip-4337) smart contract accounts (SCAs) that support programmable account behavior and advanced features such as multi-signature approvals, transaction batching, and custom security policies.
+
+See the [MetaMask Delegation Toolkit documentation](https://docs.gator.metamask.io/) for more information about delegator accounts and their capabilities.
+:::
+
+The key benefits of atomic batch transactions include:
+
+- **Fewer clicks and less friction** - Users only need to review and approve a single wallet confirmation, instead of multiple confirmations.
+  For example, users can confirm a spending cap and swap in one step instead of two.
+- **Faster completion times** - Only a single atomic transaction is confirmed onchain, instead of multiple individual transactions.
+- **Reduced gas fees** - When multiple transactions are executed atomically, users only need to pay a single gas fee.
+
+You can send batch transactions using [third-party libraries](#use-third-party-libraries)
+or [directly in your dapp](#send-batch-transactions-1).
+
+## Use third-party libraries
+
+You can send batch transactions using the following third-party libraries that support EIP-5792:
+
+- [Wagmi](https://wagmi.sh/react/api/hooks/useCapabilities)
+- [Viem](https://viem.sh/experimental/eip5792/client)
```

Comment:
5792 is no longer experimental in viem. You now have the all the 5792 methods under actions/Call bundles - https://viem.sh/docs/actions/wallet/sendCalls